src/main/java/Duke.java

```diff
@@ -1,10 +1,172 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    public static void main(String[] args) throws DukeException, IOException{
+        String pathName = "./data/duke.txt";
+        File f = new File(pathName);
+        try {
+            if(!f.exists()){
+                f.createNewFile();
+            }
+            Scanner sc = new Scanner(f);
+            ArrayList<Task> t = new ArrayList<>();
+            while (sc.hasNext()) {
+                String task = sc.nextLine();
+                String[] taskArr = task.split("//|");
+                if (taskArr[0].equals("T")) {
+                    Todo td = new Todo(taskArr[2]);
+                    if (taskArr[1].equals("1")) {
+                        td.markAsDone();
+                    }
+                    t.add(td);
+                }
+                if (taskArr[0].equals("D")) {
+                    Deadline d = new Deadline(taskArr[2], taskArr[3]);
+                    if (taskArr[1].equals("1")) {
+                        d.markAsDone();
+                    }
+                    t.add(d);
+                }
+                if (taskArr[0].equals("E")) {
+                    Event e = new Event(taskArr[2], taskArr[3]);
+                    if (taskArr[1].equals("1")) {
+                        e.markAsDone();
+                    }
+                    t.add(e);
+                }
+            }
+            int ctr = 0;
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("_______________________________________________");
```

Comment:
Can perhaps use a function to print the dividers to make the code look more readable!